ci-operator/step-registry/openstack/conf/createfips/openstack-conf-createfips-commands.sh

```diff
@@ -4,23 +4,28 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+if [[ "${CONFIG_TYPE}" == "fipless" ]]; then
+    echo "Skipping step due to CONFIG_TYPE being fipless."
+    exit 0
+fi
+
 export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml
 CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
 OPENSTACK_EXTERNAL_NETWORK="${OPENSTACK_EXTERNAL_NETWORK:-$(<"${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK")}"
 
 LB_FIP_AND_ID=$(openstack floating ip create --description $CLUSTER_NAME.api-fip $OPENSTACK_EXTERNAL_NETWORK --format value -c 'floating_ip_address' -c 'id')
-echo ${LB_FIP_AND_ID} |awk '{print $1}' > ${SHARED_DIR}/LB_FIP_IP
+echo ${LB_FIP_AND_ID} |awk '{print $1}' > ${SHARED_DIR}/API_IP
```

Comment:
@pierreprinetti the file won't be created in FIPless mode, we need to create it elsewhere as well.